test_integration/02_stake_and_mint/utils/assert_progress_mint.js

```diff
@@ -0,0 +1,210 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+const assert = require('assert');
+const BN = require('bn.js');
+
+/**
+ *  Class to assert event and balances after progress mint
+ */
+class AssertProgressMint {
+    /**
+     *
+     * @param web3 Auxiliary web3
+     * @param coGateway CoGateway contract instance.
+     * @param ostPrime OSTPrime contract instace.
+     */
+    constructor(web3, coGateway, ostPrime) {
+        this.web3 = web3;
+        this.coGateway = coGateway;
+        this.ostPrime = ostPrime;
+    }
+
+    /**
+     *  This verifies event and balances.
+     *
```

Comment:
Sometimes you have an empty line after the first JSDoc line, and sometimes not 🤔 Do we want to agree an one style? I don't mind which one.